samples/graphene-redis/Dockerfile

```
@@ -12,15 +12,26 @@ RUN make premain-graphene
 # Use with fully built graphene as build context
 # Place redis-server.manifest.template from this repo inside $GRAPHENE_DIR/Examples/redis/
 FROM ghcr.io/edgelesssys/edgelessrt-deploy:latest AS release
-RUN apt-get update && apt-get install -y build-essential autoconf gawk bison wget python3 libcurl4-openssl-dev \
-    python3-protobuf libprotobuf-c-dev protobuf-c-compiler python3-pip software-properties-common
+RUN apt-get update && apt-get install -y git meson build-essential autoconf gawk bison wget python3 libcurl4-openssl-dev \
+    python3-protobuf libprotobuf-c-dev protobuf-c-compiler python3-pip software-properties-common python3-click python3-jinja2 \
+    linux-headers-$(uname -r)
 RUN wget -qO- https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add
 RUN add-apt-repository 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main'
 RUN apt-get install -y libsgx-quote-ex-dev libsgx-aesm-launch-plugin
-RUN python3 -m pip install toml
-COPY . /graphene
-COPY --from=build-premain /premain/build/premain-graphene /graphene/Examples/redis
+RUN python3 -m pip install toml>=0.10
```

```suggestion
RUN python3 -m pip install "toml>=0.10"
```
running this without quotation marks leads to installation of `toml`, not necessarily at version `>=0.10`, with the output piped to a file called `=0.10`